sklearn/metrics/cluster/_supervised.py

```diff
@@ -138,6 +146,206 @@ def contingency_matrix(labels_true, labels_pred, *, eps=None, sparse=False):
 
 # clustering measures
 
+def pair_confusion_matrix(labels_true, labels_pred):
+    """Pair confusion matrix arising from two clusterings.
+
+    The pair confusion matrix computes a 2x2 similarity matrix between
+    two clusterings by considering all pairs of samples and counting
+    pairs that are assigned in the same or different clusters in the
+    predicted and true clusterings.
+
+    The 2x2 pair confusion matrix::
```

Comment:
Please replace this with "read more in the user guide" with the syntax used in most other API docstrings.

Reply:
Done.

Reply:
Done.